gsoc/models.py

```diff
@@ -0,0 +1,204 @@
+    orgs_can_be_edited = models.BooleanField()
+    mentor_finals_can_be_submitted = models.BooleanField()
+    slot_allocation_emails_can_be_sent = models.BooleanField()
+    projects_can_be_accepted = models.BooleanField()
+    enrollments_can_undergo_second_review = models.BooleanField()
+    orgs_can_be_accepted = models.BooleanField()
+    students_can_register = models.BooleanField()
+    second_enrollments_can_be_reviewed = models.BooleanField()
+    second_enrollment_forms_can_be_re_submitted = models.BooleanField()
+    org_members_can_edit_proposal = models.BooleanField()
+    evaluations_in_progress = models.BooleanField()
+    proposal_can_be_deleted = models.BooleanField()
+    accepted_orgs_published = models.BooleanField()
+    results_published = models.BooleanField()
+    send_assignee_emails = models.BooleanField()
+    org_members_can_view_final_proposals = models.BooleanField()
+    orgs_can_register = models.BooleanField()
+    student_second_evals_can_be_submitted = models.BooleanField()
+    enrollment_forms_can_be_submitted = models.BooleanField()
+    project_decisions_emails_can_be_sent = models.BooleanField()
+    orgs_can_see_enrollment_status = models.BooleanField()
+    proposal_in_progress = models.BooleanField()
+    orgs_have_been_accepted = models.BooleanField()
+    mentors_can_register = models.BooleanField()
+    final_evaluations_emails_can_be_sent = models.BooleanField()
+    slots_can_be_requested = models.BooleanField()
+    org_accept_reject_emails_can_be_sent = models.BooleanField()
+    org_payment_details_visible = models.BooleanField()
+    students_can_submit_proposals = models.BooleanField()
+    student_withdrawals_trigger_email = models.BooleanField()
+    enrollment_forms_can_be_re_submitted = models.BooleanField()
+    tax_forms_can_be_reviewed = models.BooleanField()
+    first_eval_emails_can_be_sent = models.BooleanField()
+    second_review_status_visible = models.BooleanField()
+    project_decisions_can_be_finalized = models.BooleanField()
+    student_home_address_is_editable = models.BooleanField()
+    program = models.ForeignKey(Program, on_delete=models.CASCADE)
+
+    def __str__(self):
+        return str(self.program)
+
+
+class ProgramStatistic(models.Model):
+    number_of_lines_of_code = models.IntegerField()
+    homepage_start_button = models.CharField(max_length=100)
+    homepage_intro_paragraph = models.TextField()
+    number_of_mentors = models.IntegerField()
+    number_of_student_and_mentor_countries = models.IntegerField()
+    number_of_years = models.IntegerField()
+    homepage_orgs_paragraph = models.TextField()
+    homepage_students_paragraph = models.TextField()
+    number_of_organizations = models.IntegerField()
+    number_of_student_countries = models.IntegerField()
+    program = models.ForeignKey(Program, on_delete=models.CASCADE)
+
+    def __str__(self):
+        return self.homepage_start_button
```

Comment:
there is no good attribute to use for the `str()`.  can we use the program year ; it is the foreign key ??